src/libraries/Microsoft.PowerFx.Core/Binding/INameResolver.cs

```diff
@@ -36,6 +36,8 @@ internal interface INameResolver
 
         IEnumerable<TexlFunction> Functions { get; }
 
+        bool SuggestUnqualifiedEnums { get; }
```

Comment:
>bool [](http://example.com/codeflow?start=8&length=4)

We should add a comment like:

This advertises whether the INameResolver instance will suggest unqualified enums ("Hours") or only qualified enums ("TimeUnit.Hours"). This must be consistent with how the other Lookup functions behave. 

Intellisense can use this when suggesting completion options.

Reply:
Done